parkings/api/enforcement/valid_parking.py

```diff
@@ -41,6 +42,13 @@ def to_representation(self, instance):
         return representation
 
 
+class ValidParkingAndPermitSerializer(ValidParkingSerializer, serializers.Serializer):
```

Comment:
Perhaps the new endpoint changes should be in a different commit, as they are unrelated to the are field changes.